javaparser-core/src/main/java/com/github/javaparser/ast/NodeList.java

```diff
@@ -472,4 +473,9 @@ private void setAsParentNodeOf(Node childNode) {
             childNode.setParentNode(getParentNodeForChildren());
         }
     }
+
+    @Override
+    public String toString() {
+        return innerList.stream().map(Node::toString).collect(Collectors.joining());
```

Comment:
should this be `joining(", ")` ?

Reply:
I would also consider adding some delimiters, useful in particular for empty lists. For example: "[" and "]"